src/Config/permissions.rs

```diff
@@ -106,17 +106,11 @@ impl BridgePermissions {
         service: String,
         permission: String,
     ) -> napi::Result<bool> {
-        let parts: Vec<&str> = mxid.split(':').collect();
-        let domain: String;
+        let server_name = mxid.split_once(":").unwrap().1.to_owned();
```

Comment:
Beware of this change:
The new `.unwrap()` will throw, if the `mxid` is invalid and doesn't contain a colon.